### PR TITLE
Request totals for dashboard v2

### LINKED ISSUE #193: [dashboard] write a banner component
Part of #183

### PR DESCRIPTION
**Pull request by esbie**
Implements a module that displays totals of requests/connections being made. There's a lot of metrics manipulation here that I think could be simplified going forwards

screen shot 2016-03-29 at 3 27 09 pm

Fixes #193

### CODE REVIEW

**Comment by esbie** [comment]
realizing I made a fatal mistake by not naming this module TotalRequestsLive

File: linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/request_totals.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  ];
+
+
+  /*
+   * Maps metric definitions of a specific type to metric ids
+   * using provided prefixes and definition suffixes
+   */
+  function toMetricIds(objs, metricType) {
+    return _(objs).map(function(obj) {
+      var metrics = _.filter(metricDefinitions, ["metricType", metricType]);
```
Comment (rmars) [inline linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/request_totals.js]: This can go outside the map()

**Comment by rmars** [comment]
:star: :package: :rabbit:  I like this branch a lot

**Comment by klingerf** [comment]
:star: :panda_face:

**Comment by siggy** [comment]
:star: :+1:

**Pull request merged**